### PR TITLE
Show Download all as ZIP button only when all conditions are met 

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java [hunk 1]
```diff
@@ -151,6 +152,10 @@ public class ItemServiceImpl extends DSpaceObjectServiceImpl<Item> implements It
     @Autowired(required = true)
     private EntityTypeService entityTypeService;
 
+
+    @Autowired(required = true)
+    protected MetadataValueDAO metadataValueDAO;
```
Comment (vidiecan) [inline dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java:157]: why?

File: dspace-api/src/test/java/org/dspace/content/VersioningWithRelationshipsTest.java [hunk 2]
```diff
@@ -1,4203 +0,0 @@
-/**
- * The contents of this file are subject to the license and copyright
```
Comment (vidiecan) [inline dspace-api/src/test/java/org/dspace/content/VersioningWithRelationshipsTest.java:2]: why did you remove it?

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java (hunk 1 above)
Comment (milanmajchrak) [reply dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java:157]: Because it is duplicate of `VersioningWithRelationshipsIT.java`

File: dspace-api/src/test/java/org/dspace/content/VersioningWithRelationshipsTest.java (hunk 2 above)
Comment (milanmajchrak) [reply dspace-api/src/test/java/org/dspace/content/VersioningWithRelationshipsTest.java:2]: Because it is duplicate of VersioningWithRelationshipsIT.java

File: dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java (hunk 1 above)
Comment (milanmajchrak) [reply dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java]: That's mistake, thank you

**Review by vidiecan — APPROVED** [review]
